docs/types/scalars.md

```diff
@@ -117,3 +117,49 @@ result = schema.execute_sync("{ base64 }")
 
 assert results.data  == {"base64": "aGk="}
 ```
+
+## Example JSONScalar
+
+```
+import json
+from typing import Any, NewType
+
+import strawberry
+
+JSONScalar = strawberry.scalar(
+    NewType("JSONScalar", Any),
+    serialize=lambda v: v,
+    parse_value=lambda v: json.loads(v),
+    description="The GenericScalar scalar type represents a generic GraphQL scalar value that could be: List or Object."
+)
+
+```
+
+Usage
+```
+@strawberry.type
+class Query:
+    @strawberry.field
+    def data(self, info) -> GenericScalar:
+        return {"hello": {"a": 1}, "someNumbers": [1, 2, 3]}
+
+```
+
+Query
+```
+query ExampleDataQuery {
+  data
+}
+```
+
+Returns
+```
+{
+  "data": {
+    "hello": {
+      "a": 1
+    },
+    "someNumbers": [1,2, 3]
+  }
+}
+```
```

Comment:
In the docs we have a custom language that presents queries and responses side by side (for example https://strawberry.rocks/docs/guides/errors#graphql-validation-errors). So this code example can become:

```suggestion
```graphql+response
query ExampleDataQuery {
  data
}
---
{
  "data": {
    "hello": {
      "a": 1
    },
    "someNumbers": [1, 2, 3]
  }
}
```
```